Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
Basically, as long as you're not relying on Homebrew being able to write to that prefix during install or test, that's fine to use the default.